### PR TITLE
Fix update global formation template webhook

### PR DESCRIPTION
**Pull request by ivantenevvasilev**
**Description**
`updateWebhook` on webhooks that reference a global formation template failed, as it was calling the tenant-isolated version with an empty tenant resulting in an invalid SQL query.

Changes proposed in this pull request:
-  adapt repository to use global updater for global formation templates
- add unit tests
- add e2e test that updates a webhook references a global formation template

**Related issue(s)**
- #2840

**Pull Request status**

- [x] Implementation
- [x] Unit tests
- [x] Integration tests
- [x] `chart/compass/values.yaml` is updated 
- [x] [N/A] Mocks are regenerated, using the automated script

### CODE REVIEW

File: tests/director/tests/formation_templates_test.go [hunk 1]
```diff
@@ -332,6 +332,21 @@ func TestModifyFormationTemplateWebhooks(t *testing.T) {
 		assertions.AssertWebhooks(t, []*graphql.WebhookInput{webhookInput}, []graphql.Webhook{*updatedFormationTemplate.Webhooks[0]})
 	})
 
+	urlUpdated := "https://test.com"
+	t.Run("Update formation template webhook", func(t *testing.T) {
```
Comment (PetarTodorovv) [inline tests/director/tests/formation_templates_test.go:336]: **_minor:_** What do you think for `Update formation template webhook globally`?

Also, there were a TODOs in the formation_api_test.go file related to the update FT webhook globally. I'm not sure if they will be adapter as part of this PR or in a separate one. Just a heads-up :)

**Review by PetarTodorovv — APPROVED** [review]

File: tests/director/tests/formation_templates_test.go (hunk 1 above)
Comment (ivantenevvasilev) [reply tests/director/tests/formation_templates_test.go:336]: Nice catch, I didn't realize the other PR was merged. I'll adapt them here.

File: tests/director/tests/formation_api_test.go [hunk 2]
```diff
@@ -2025,14 +2025,11 @@ func TestFormationNotificationsWithApplicationOnlyParticipants(t *testing.T) {
 		assertFormationStatus(t, ctx, tnt, formation.ID, graphql.FormationStatus{Condition: graphql.FormationStatusConditionInProgress, Errors: nil})
 
 		urlTemplateThatFailsOnce := "{\\\"path\\\":\\\"" + conf.ExternalServicesMockMtlsSecuredURL + "/v1/businessIntegration/async-fail-once/{{.Formation.ID}}\\\",\\\"method\\\":\\\"{{if eq .Operation \\\"createFormation\\\"}}POST{{else}}DELETE{{end}}\\\"}"
-
-		// TODO::Change with updateWebhook when it is fixed for global FT
 		webhookThatFailsOnceInput := fixtures.FixFormationNotificationWebhookInput(formationTemplateWebhookType, formationTemplateWebhookMode, urlTemplateThatFailsOnce, inputTemplateFormation, outputTemplateFormation)
 
 		t.Logf("Add webhook with type %q and mode: %q to formation template with ID: %q", formationTemplateWebhookType, formationTemplateWebhookMode, ft.ID)
```
Comment (la4ezar) [inline tests/director/tests/formation_api_test.go:2030]: Change the log message because now we are updating the webhook not adding.

**Comment by ivantenevvasilev** [comment]
reset-vm

**Comment by ivantenevvasilev** [comment]
reset-vm

**Review by PetarTodorovv — APPROVED** [review]

**Review by PetarTodorovv — APPROVED** [review]

**Pull request merged**